packages/binding-http/test/http-server-test.ts

```diff
@@ -213,6 +213,36 @@ class HttpServerTest {
         return httpServer.stop();
     }
 
+    @test async "should return 204 when action has not output"() {
+        const httpServer = new HttpServer({ port: 0 });
+
+        await httpServer.start(null);
+
+        const testThing = new ExposedThing(null, {
+            title: "Test",
+            actions: {
+                noOutput: {
+                    output: { type: "string" },
+                    forms: [],
+                },
+            },
+        });
+
+        testThing.setActionHandler("noOutput", async () => undefined);
+
+        await httpServer.expose(testThing);
+
+        const uri = `http://localhost:${httpServer.getPort()}/test/`;
+
+        debug(`Testing ${uri}`);
+
+        const resp = await fetch(uri + "actions/noOutput", { method: "POST" });
+
+        expect(resp.status).to.equal(204);
```

Comment:
Just for the sake of completeness. Does it make sense to add

`expect(resp.status).to.equal(200);`

to one of the other action tests?

Reply:
Yes, let me see if there is a place where it fits.

Reply:
tackled in https://github.com/eclipse-thingweb/node-wot/pull/1042/commits/479d66d543d7d5b6878939a8516467fd8daff790 . Feel free to merge it if it is ok.